source/features/list-prs-for-file.tsx

```diff
@@ -162,7 +162,7 @@ void features.add(import.meta.url, {
 		pageDetect.isEditingFile,
 		pageDetect.isSingleFile,
 	],
-	deduplicate: '.rgh-list-prs-for-file', // #3945
```

Comment:
The dropdown was missing that class, but the button had it.

Reply:
Doesn't this reintroduce https://github.com/refined-github/refined-github/issues/3945?